examples/vm-joined-to-active-directory/README.md

```diff
@@ -0,0 +1,87 @@
+# Example: Joining a Windows Machine to an Active Directory Domain
+
+This example creates an Active Directory Domain, a Windows Client; to demonstrate how to bind a Windows Client to an Active Directory Domain using a Virtual Machine Extension in Terraform (using [the `azurerm_virtual_machine_extension` resource](https://www.terraform.io/docs/providers/azurerm/r/virtual_machine_extension.html)).
+
+This example is built around [the Virtual Machine Extension](https://www.terraform.io/docs/providers/azurerm/r/virtual_machine_extension.html) found in the `windows-client` module (documented below) - which demonstrates binding a Windows Virtual Machine to an Active Directory Domain. For the purposes of this example we create an Active Directory Domain, since it's easier to demonstrate - however you can achieve the same thing with an existing Active Directory Domain.
+
+## Notes
+
+- This is intended as an example of binding machines to an Active Directory Domain, and **we'd not recommend using it in production as-is** as the configuration has been simplified for example purposes, e.g.:
```

Comment:
nitpick:

> it is not recommended for production use

feels a bit stronger to me than "we'd not recommend"

Reply:
fixed